app/frontend/components/common/asset.tsx

```diff
@@ -20,7 +20,7 @@ export const LinkToAsset = ({policyIdHex, assetNameHex}: LinkToAssetProps) => (
 )
 
 export type FormattedAssetItemProps = Token & {
-  fingerprint: string
+  fingerprint: string | null
```

Comment:
assets should always have a fingerprint. If this can be null, I'd rather check the root cause and assert there

Reply:
I see it comes from DropdownAssetItem which indeed can be ADA as well, ok nevermind